lib/modules/vyper/index.js

```diff
@@ -36,6 +36,14 @@ class Vyper {
       return cb();
     }
     self.logger.info(__("compiling Vyper contracts") + "...");
+    
+    const vyper = shelljs.which('vyper');
+    if (!vyper) {
+      self.logger.warn(__('%s is not installed on your machine', 'Vyper'));
+      self.logger.info(__('You can install it by visiting: %s', 'https://vyper.readthedocs.io/en/latest/installing-vyper.html'));
+      return cb();
+    }
```

Comment:
Nice catch! You should move it even before "compiling...", that way the user doesn't get a weird idea that it's actually compiling.